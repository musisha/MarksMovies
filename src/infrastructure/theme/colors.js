export const colors = {
  ui: {
    primary: "#172731",
    secondary: "#fff",
    disabled: "#DEDEDE",
    error: "#D0421B",
    success: "#138000",
  },
  bg: {
    primary: "#172731",
    secondary: "#fff",
    tertiary: "#808080",
  },
  text: {
    primary: "#fff",
    secondary: "#757575",
    disabled: "#9C9C9C",
    inverse: "#FFFFFF",
    error: "#D0421B",
    success: "#138000",
  },
};
