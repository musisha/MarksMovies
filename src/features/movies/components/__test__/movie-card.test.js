import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import { MovieCard } from "../MovieCard/moviecard.component";
import { theme } from "../../../../infrastructure/theme";

describe("<MovieCard>", () => {
  it("render without issues", () => {
    const wrapper = shallow(<MovieCard theme={theme} />);
    expect(wrapper.length).toBe(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
