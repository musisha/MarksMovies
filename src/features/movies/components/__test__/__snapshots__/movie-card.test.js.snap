// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCard> it works 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "#252250",
        "borderColor": "#fff",
        "borderRadius": 4,
        "borderStyle": "solid",
        "borderWidth": 1,
        "elevation": 1,
        "height": 200,
        "marginBottom": 10,
        "marginLeft": 10,
        "marginRight": 10,
        "marginTop": 10,
        "shadowColor": "#000000",
        "shadowOffset": Object {
          "height": 0.5,
          "width": 0,
        },
        "shadowOpacity": 0.24,
        "shadowRadius": 0.75,
        "width": 150,
      }
    }
  >
    <View
      accessible={true}
      focusable={false}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "flexGrow": 1,
          "flexShrink": 1,
        }
      }
    >
      <View
        accessibilityIgnoresInvertColors={true}
        style={
          Array [
            Object {
              "alignItems": "center",
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
              "height": "50%",
              "justifyContent": "flex-end",
              "resizeMode": "contain",
              "width": "100%",
            },
          ]
        }
      >
        <Image
          index={0}
          siblings={
            Array [
              "Styled(ImageBackground)",
            ]
          }
          source={
            Object {
              "uri": "https://lh3.googleusercontent.com/proxy/BOwUj6M_wMToA6jVxqt9ATQpbAk2uM50NmjLlXwpL9rmPOlXj6vcs05Qb5Laslx-MDIGSItnY0uqd5nlcJd1V-0if7VEE3yDX_ONZh3ZgISgIUojhmzxrjKa_kFGvVyl4lxJ583nVPdXroOuHLcS996vWvJk0LL9JIldrrLLKFBh4XXfcRHTQZVzJC8jo9o-waPtjQVsvkGU",
            }
          }
          style={
            Array [
              Object {
                "bottom": 0,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "height": undefined,
                "width": undefined,
              },
              undefined,
            ]
          }
          total={1}
        />
      </View>
    </View>
  </View>
</View>
`;

exports[`<MovieCard> render without issues 1`] = `
<Fragment>
  <ForwardRef
    onPress={[Function]}
  >
    <Styled(withTheme(Card))>
      <Styled(ImageBackground)
        source={
          Object {
            "uri": "https://lh3.googleusercontent.com/proxy/BOwUj6M_wMToA6jVxqt9ATQpbAk2uM50NmjLlXwpL9rmPOlXj6vcs05Qb5Laslx-MDIGSItnY0uqd5nlcJd1V-0if7VEE3yDX_ONZh3ZgISgIUojhmzxrjKa_kFGvVyl4lxJ583nVPdXroOuHLcS996vWvJk0LL9JIldrrLLKFBh4XXfcRHTQZVzJC8jo9o-waPtjQVsvkGU",
          }
        }
      />
    </Styled(withTheme(Card))>
  </ForwardRef>
</Fragment>
`;
